homeassistant/components/tmb/sensor.py

```diff
@@ -0,0 +1,113 @@
+"""Support for TMB (Transports Metropolitans de Barcelona) Barcelona public transport."""
+from datetime import timedelta
+
+from tmb import IBus
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME, ATTR_ATTRIBUTION
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+ATTRIBUTION = "Data provided by Transport Metropolitans de Barcelona"
+
+ICON = "mdi:bus-clock"
+
+CONF_APP_ID = "app_id"
+CONF_APP_KEY = "app_key"
+CONF_STOP = "stop"
```

Comment:
Maybe CONF_BUS_STOP may be more appropriate and avoid confusion with the meaning of the variable.

Reply:
Make sense, changed!